index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html>
+
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Aline's page</title>
+    <link rel="stylesheet" type="text/css" href="style.css">
+  <body>
+    <!-- 
+      Write your code here inside the body to make it appear in the browser. 
+      Below is an example on how to use an image tag. Copy it and add it in the body, but outside the comment tag if you want to see how it looks like on the website. Then you can change the size and other styling for it in the css file.
+
+      <img src="example-recipe-background.png" alt="Banana Bread">
+
+    -->
+    <h1 class="top-heading">Aline Schlunegger</h1>
+     <h2 class="second-heading">Film-Online-Marketing-Redaktion 🎬✏️💬📱</h2>
+     <div class="light-coral-box">
+      <p class="paragraphs">🎬 Film - und alle anderen Formate, die bewegen.<p>
+<p class="paragraphs">Erfahrungen in: </p>
+<p class="paragraphs">🎟 Filmmarketing</p>
+<p class="paragraphs">🖋 Filmkritik</p>
+<p class="paragraphs">📺 Vertrieb und Verleih</p>
+<p class="paragraphs">🌎 International und national</p>
+<p class="paragraphs">#️⃣🎞 Online und offline</p>
+<p class="paragraphs">👩🏻‍🎓 funderiertes kultur- und filmwissenschaftliches Wissen.</p>
+</div>
```

Comment:
Something funny 🤡 is going on here with the indentation. Remember that if there's a new block, then you indent one more level. At Technigo, we use 2 spaces for indentation. Also: all the elements that are siblings should be in the same level. 🧑‍🤝‍🧑